lib/widgets/message_list.dart

```diff
@@ -396,7 +397,11 @@ class MarkAsReadWidget extends StatelessWidget {
             child: FilledButton.icon(
               style: FilledButton.styleFrom(
                 padding: const EdgeInsets.all(10),
-                textStyle: const TextStyle(fontSize: 18, fontWeight: FontWeight.w200),
+                textStyle: const TextStyle(
+                  fontFamily: 'Source Sans 3',
+                  fontSize: 18,
+                  fontWeight: FontWeight.w400,
```

Comment:
The `fontWeight: FontWeight.w400,` is redundant with `.merge(weightVariableTextStyle(context))`.